src/udiskslinuxdrive.c

```diff
@@ -50,6 +51,71 @@
 #include "udisksdaemonutil.h"
 #include "udiskslinuxdevice.h"
 
+static const gchar *
+_get_blk_path (UDisksDrive *drv, GDBusMethodInvocation *invocation);
```

Comment:
Just a personal naming habit, not a big deal. Will remove the leading underscore.